frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.tsx

```diff
@@ -65,7 +65,7 @@ export const PageConfigPanel = () => {
         }}
       />
       {layoutIsSelected && (
-        <>
+        <span key={selectedFormLayoutName}>
```

Comment:
Since `span` is reserved for [phrasing content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#phrasing_content) I think we should avoid using it here. But I think we can still use a [React Fragment](https://react.dev/reference/react/Fragment) if we define it like this: 
```suggestion
        <Fragment key={selectedFormLayoutName}>
```

Reply:
Nice to know 🙂 Thx!